src/Simmy/Simmy.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.1;netstandard2.0</TargetFrameworks>
+    <AssemblyName>Simmy</AssemblyName>
+    <RootNamespace>Simmy</RootNamespace>
+    <Version>0.0.1</Version>
+    <AssemblyVersion>0.0.1</AssemblyVersion>
+    <FileVersion>0.0.1</FileVersion>
+    <InformationalVersion>0.0.1</InformationalVersion>
+    <PackageVersion>0.0.1</PackageVersion>
+    <Company>App vNext</Company>
+    <Copyright>Copyright (c) 2019, App vNext</Copyright>
+    <Description>Simmy is a chaos-engineering and fault-injection tool, integrating with the Polly resilience project for .NET</Description>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Authors>App vNext</Authors>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DebugType>full</DebugType>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard1.1' ">
+    <NetStandardImplicitPackageVersion>1.6.1</NetStandardImplicitPackageVersion>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
```

Comment:
Let's remove this inclusion of `System.Runtime.Serialization.Formatters`.  For Polly, it was there because Polly defines some custom exceptions, and this lets/helps allow binary serialization for them (which .Net Standard 1.1 didn't support).

However, Simmy doesn't define any custom exceptions (at least so far), so we don't need it.

Reply:
Got it!